wi

```diff
@@ -22,30 +22,62 @@ import asyncio
 import os
 import sys
 import tempfile
+import argparse
 
 import aiohttp
 from aiohttp.client_exceptions import ContentTypeError
 from wheel.install import WheelFile
 
+from html.parser import HTMLParser
+
+
+class PyPIParser(HTMLParser):
+
+    def __init__(self, *args, **kwargs):
+        # Format is
+        # <a href='url'>filename</a>
+        self.all_data = []
+        self.current = None
+        super().__init__(*args, **kwargs)
+
+    def handle_starttag(self, tag, attrs):
+        # Keep only references in <a> tags
+        if tag == 'a':
+            self.current = {
+                'href': [value for attr, value in attrs if attr == 'href'][0],
+            }
+        else:
+            self.current = None
+
+    def handle_endtag(self, tag):
+        if self.current:
+            self.all_data.append(self.current)
+
+    def handle_data(self, data):
+        if self.current:
+            self.current['filename'] = data
+
+
 PACKAGES_REPO_URL = 'https://pypi.python.org/pypi'
 
```

Comment:
I think we can remove this constant now.

Reply:
Removed